erpnext/agriculture/doctype/crop_cycle/test_crop_cycle.py

```diff
@@ -3,66 +3,72 @@
 # See license.txt
 from __future__ import unicode_literals
 
-import frappe
 import unittest
 
-test_dependencies = ["Crop", "Fertilizer", "Land Unit", "Disease"]
+import frappe
+from frappe.utils import datetime
+
+test_dependencies = ["Crop", "Fertilizer", "Location", "Disease"]
+
 
 class TestCropCycle(unittest.TestCase):
 	def test_crop_cycle_creation(self):
 		cycle = frappe.get_doc('Crop Cycle', 'Basil from seed 2017')
-		self.assertEqual(frappe.db.exists('Crop Cycle', 'Basil from seed 2017'), 'Basil from seed 2017')
+		self.assertTrue(frappe.db.exists('Crop Cycle', 'Basil from seed 2017'))
 
-		# check if the tasks were created 
+		# check if the tasks were created
 		self.assertEqual(check_task_creation(), True)
 		self.assertEqual(check_project_creation(), True)
 
+
 def check_task_creation():
 	all_task_dict = {
 		"Survey and find the aphid locations": {
-			"exp_start_date": frappe.utils.datetime.date(2017,11,21),
-			"exp_end_date": frappe.utils.datetime.date(2017,11,22)
+			"exp_start_date": datetime.date(2017, 11, 21),
```

Comment:
any reason for not using `frappe.utils` ??

Reply:
Not functionally, this was just a sanity edit. :sweat_smile: